frontend/src/routes/candidate/(protected)/profile/+page.svelte

```diff
@@ -90,7 +96,7 @@
     <FieldGroup>
       <label for="message" class={disclaimerClass} slot="header"
         >{$_('candidateApp.basicInfo.electionManifesto')}</label>
-      <textarea id="message" rows="4" class="w-full resize-none bg-base-100 p-6 !outline-none" />
+      <textarea id="message" rows="4" class="bg-base-100 w-full resize-none p-6 !outline-none" />
```

Comment:
For me, Prettier formats these in the way it was originally. I suggest you try reinstalling the npm dependencies in the frontend and format the files again. If that doesn't work, then I no longer know what causes this issue where it formats differently on different environments :shrug: